client/components/navigation-header/index.tsx

```diff
@@ -7,10 +8,6 @@ import FormattedHeader from '../formatted-header';
 import './style.scss';
 
 const Container = styled.div`
-	@media ( max-width: 660px ) {
-		min-height: 60px;
-	}
```

Comment:
@candy02058912 do you recall if this was for handling the screen options tab or was there another purpose? It was adding to the overall height leaving a lot of whitespace so I dropped it from here. If it's needed for something else we could reduce some of the padding elsewhere to compensate.

Reply:
I directly copied it from `FixedNavigationHeader`, so I'm not sure where the effects will kick in. We can remove it and add it back if needed.